### DISCUSSION TITLE
Is there an official way to do a ping in MicroPython?  Is there a better variant of uping if that is the way to go?

### DISCUSSION THREAD

**Discussion by Jibun-no-Kage** [micropython/micropython#15978] [General]
Is there an official way to do a ping in MicroPython?  Is there a better variant of uping if that is the way to go?  I mean I found several variants of uping in github alone, but I have no clue which one is (more) legit than the others?  Is there a better more native way in MicroPython to do a simple ping?

**Comment by peterhinch** [comment]
[This thread](https://forum.micropython.org/viewtopic.php?f=15&t=5287) describes a version of uPing which was thoroughly tested.

**Comment by Jibun-no-Kage** [comment]
Thanks!
